hbase-client/src/main/java/org/apache/hadoop/hbase/client/CancellableRegionServerCallable.java

```diff
@@ -62,7 +61,7 @@ public T call(int operationTimeout) throws IOException {
     int remainingTime = tracker.getRemainingTime(operationTimeout);
     if (remainingTime <= 1) {
       // "1" is a special return value in RetryingTimeTracker, see its implementation.
-      throw new DoNotRetryIOException("Operation rpcTimeout");
+      throw new OperationTimeoutExceededException("?? What additional information would we want in this message ??");
```

Comment:
@bbeaudreault - repeating `Operation rpcTimeout` in the exception message felt redundant, any ideas on what might be helpful to see here?

Reply:
hmm we could either go with nothing, or something like "Timeout exceeded before call began. Slow meta or timeout too short?".  Haha probably could be improved, but something like that?